lib/streamlit/DeltaGenerator.py

```diff
@@ -2361,6 +2361,10 @@ def date_input(self, element, label, value=None, key=None):
         value : datetime.date/datetime.datetime
             The value of this widget when it first renders. This will be
             cast to str internally. Defaults to today.
+        min_date : datetime.date/datetime.datetime
+            A min date that is selectable. Defaults to 1970/1/1
```

Comment:
Can we make the min date be `datetime.datetime.min`, which is 1/1/1?
This is supported by JS too (actually JS also supports negative years, down to many hundreds of years BCE). It looks like BaseWeb only goes to 1/1/1, though, but that's enough to match Python.

Reply:
If I putt 1/1/1 the component crashes due to a memory problem that's why I set 1970